components/TopBarProfileMenu.js

```diff
@@ -19,6 +20,24 @@ import LoginBtn from './LoginBtn';
 import StyledLink from './StyledLink';
 import { P } from './Text';
 import { withUser } from './UserProvider';
+import StyledRoundButton from './StyledRoundButton';
+import { Plus } from '@styled-icons/boxicons-regular';
+import StyledHr from './StyledHr';
+import styled from 'styled-components';
+
+const CollectiveListItem = styled(ListItem)`
+  @media (hover: hover) {
+    :hover .SettingsCogIcon {
+      opacity: 1;
+    }
+  }
+
+  @media (hover: none) {
+    .SettingsCogIcon {
+      opacity: 1;
+    }
+  }
```

Comment:
Adding a `className` is not the way to go with styled-components. I suggest looking at https://styled-components.com/docs/advanced#referring-to-other-components to learn more about the best practice for this.

Here I think the easiest solution is just to do:
```suggestion
  @media (hover: hover) {
    :hover svg {
      opacity: 1;
    }
  }

  @media (hover: none) {
    svg {
      opacity: 1;
    }
  }
```

Reply:
Done. Thanks for pointing it out and sorry, I am still learning styled-components; and this way of doing things is new to me. :smile: